tests/spread/core24/cross-compile/snapcraft.yaml

```diff
@@ -0,0 +1,49 @@
+name: curl-consumer
```

Comment:
@medubelko material for future howto

Reply:
Yes, this would be a good replacement snapcraft.yaml for https://snapcraft.io/docs/cross-compile-an-autotools-project